src/core/migrations/0089_update_setting_types.py

```diff
@@ -0,0 +1,95 @@
+    ('email_subject', 'subject_review_accept_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_reviewer_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_assignment', 'text', 'char'),
+    ('email_subject', 'subject_review_complete_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_complete_reviewer_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_accept', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_decline', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_undecline', 'text', 'char'),
+    ('email_subject', 'subject_review_decline_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_request_sent', 'text', 'char'),
+    ('email_subject', 'subject_review_withdrawl', 'text', 'char'),
+    ('email_subject', 'subject_revision_digest', 'text', 'char'),
+    ('email_subject', 'subject_section_editor_pub_notification', 'text', 'char'),
+    ('email_subject', 'subject_submission_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_thank_proofreaders_and_typesetters', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_complete_notification', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_corrections_complete', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_notification', 'text', 'char'),
+    ('email_subject', 'subject_typeset_ack', 'text', 'char'),
+    ('email_subject', 'subject_typeset_deleted', 'text', 'char'),
+    ('email_subject', 'subject_typeset_reopened', 'text', 'char'),
+    ('email_subject', 'subject_user_email_change', 'text', 'char'),
+    ('general', 'matromo_tracking_code', 'text', 'char'),
+    ('email_subject', 'subject_notify_correction_cancelled', 'text', 'char'),
+    ('general', 'copyright_submission_label', 'char', 'mini-html'),
+    ('email_subject', 'subject_accepted_review_reminder', 'text', 'char'),
+    ('email_subject', 'subject_default_review_reminder', 'text', 'char'),
+    ('email_subject', 'subject_notify_se_draft_declined', 'rich-text', 'char'),
+    ('crossref', 'crossref_date_suffix', 'text', 'char'),
+    ('general', 'default_journal_language', 'text', 'char'),
+    ('email_subject', 'subject_revisions_complete_receipt', 'text', 'char'),
+    ('email_subject', 'subject_author_copyedit_deleted', 'text', 'char'),
+]
+
+def update_text_and_char_setting_types(apps, schema_editor):
+    """ Standardizes use of text and char setting types"""
+
+    Setting = apps.get_model('core', 'Setting')
+    for group, name, old_type, new_type in SETTINGS_WITH_OLD_AND_NEW_TYPES:
+        update_setting_types(Setting, group, name, old_type, new_type)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0088_fix_untyped_galleys'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            update_text_and_char_setting_types,
+            reverse_code=migrations.RunPython.noop
```

Comment:
Can we have another migration that replaces `"&amp;"` for `"&"` on journal titles? Just so that we self-heal any journals that have been edited since.